docs/user-guides/manual-setup.md

```diff
@@ -0,0 +1,162 @@
+* cross-chain, ecdsa scheme key, with "key-type" `crch`
+
+For each of these keys commands that are two commands invoked by wizard.
+The first one is
+```bash
+<node-executable> key generate --scheme <scheme> --output-type json
+```
+, that has `secretPhrase` and `publicKey` fields in the output.
+Please do save these outputs as they are useful in the subsequent steps.
+
+The second one is
+```bash
+<node-executable> key insert --keystore-path <base-path>/keystore --scheme <scheme> --key-type <key-type> --suri <secretPhrase>
+```
+that stores the generated key in the keystore.
+
+Commands above serve as recipe for generating and inserting required key in the keystore.
+Will there be a need for key of different schema or type or in different location, it should now be easy to get it.
+
+The wizard can also generate node network key, using following commands
+```bash
+mkdir -p <base-path>/network
+mkdir key generate-node-key --file <base-path>/network/secret_ed25519
+```
+
+Please note that running node will require using `--base-path`, `--keystore-path` and `--node-key-file` to match the layout used above.
+
+## Prepare configuration
+
+`<node-executable> wizards prepare-configuration` sets up partner chain governance on the main chain and computes additional parameters that are used in later steps of the setup.
+
+### Establishing partner chain on Cardano
+
+Each partner chain is identified by the Genesis UTXO and it has some governance.
+Use `<node-executable> smart-contracts governance init -c <GENESIS_UTXO> -g <GOVERNANCE_AUTHORITY> -t <THRESHOLD> -k <PAYMENT_KEY_FILE>`
+command to spend the UTXO that will become the Genesis UTXO of your chain.
+The Genesis UTXO has to be spendable by the payment key.
+This command initializes governance of the chain as M of N multisig parametrized by GOVERNANCE_AUTHORITY and THRESHOLD parameters.
+
+Note: for more details please read [smart-contracts commands documentation](../../toolkit/smart-contracts/commands/readme.md).
+
+### Discover smart-contracts addresses and script hashes
+
+Use `<node-executable> smart-contracts get-scripts -c <GENESIS_UTXO>` to obtain Cardano addresses and scripts hashes.
+This guide assumes that the output is saved to `addresses.json` file.
+
+### Establish bootnodes
+
+Wizard perpares one bootnode address that is derived from the generated node network key and user input.
+This guide does not cover bootnodes as not partner chains specific topic.
+Partner chains do not add any requirements nor capabilities in regards to bootnodes.
+
+## Create chain-spec file
+
+`<node-executable> wizards create-chain-spec` uses data from the previous step to generate chain-spec file.
+It is important to understand that most of chain-spec file generation is delegated to `<node-exectuable>` itself.
+Specifically, wizard sets some specific environment variables and assumes that `<node-exectuable> build-spec` will use and that it will not require any other variables.
```

Comment:
```suggestion
Specifically, the wizard sets some specific environment variables and assumes that `<node-executable> build-spec` will use and that it will not require any other variables.
```